experimental/teams-tabs/javascript_nodejs/pages/index.html

```diff
@@ -0,0 +1,49 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Welcome to Teams Tabs</title>
+    <link rel="stylesheet" href="/css/msteams-16.css" />
+    <link rel="stylesheet" href="/css/Custom.css" />
+    <link rel="stylesheet" href="/css/Site.css" />
+</head>
+<body class="theme-light">
+    <div class="surface">
+        <div class="panel">
+
+<html>
+<head>
+    <style>
+        body {
+            margin: 0;
+            padding-left: 4px;
+            padding-right: 4px;
+        }
+    </style>
+    <title>Welcome to Teams Tabs</title>
+</head>
+<body>
+    <header class="l-header">
+        <div class="logo">
+            <img src="https://static2.sharepointonline.com/files/fabric/assets/brand-icons/product-fluent/svg/teams_48x1.svg"
+                    class="logo">
+        </div>
+        <div class="l-title">
+            <h1>Welcome to Teams <em>Tabs</em></h1>
+        </div>
+    </header>
+    <article class="l-article">
+        <p>Generate a Microsoft Teams application.</p>
+    
+        <p>
+            Add the Bot to Microsoft Teams:
+                
+            <a href='https://teams.microsoft.com/l/chat/0/0?users=28:"@Model.MicrosoftAppId"'><img height="30" width="113" src='https://dev.botframework.com/Client/Images/Add-To-MSTeams-Buttons.png'></a>
```

Comment:
I believe this is the syntax for templating in .NET, will this work the same in Node?